src/databricks/labs/remorph/coverage/remorph_snow_transpilation_coverage.py

```diff
@@ -9,6 +9,10 @@
     input_dir = commons.get_env_var("INPUT_DIR", required=True)
     output_dir = commons.get_env_var("OUTPUT_DIR", required=True)
 
+    # Ensure input_dir and output_dir are not None
+    input_dir = input_dir if input_dir is not None else ""
```

Comment:
Do error handling here, instead of pushing the input as string.

Reply:
```
if not input_dir:
        raise ValueError("Environment variable `INPUT_DIR` is required")
    if not output_dir:
        raise ValueError("Environment variable `OUTPUT_DIR` is required")
```

Handled in this way